src/test/java/org/gridsuite/network/map/NetworkMapControllerTest.java

```diff
@@ -299,6 +287,12 @@ public void setUp() {
                 .setConverterStationId2("LCC2")
                 .add();
 
+        HvdcAngleDroopActivePowerControl hvdcAngleDroopActivePowerControl = new HvdcAngleDroopActivePowerControlImpl((HvdcLineImpl) hvdcLine, 190F, 180F, true);
+        HvdcOperatorActivePowerRange hvdcOperatorActivePowerRange = new HvdcOperatorActivePowerRangeImpl((HvdcLineImpl) hvdcLine, 1000F, 900F);
+
+        hvdcLine.addExtension(HvdcOperatorActivePowerRange.class, hvdcOperatorActivePowerRange);
+        hvdcLine.addExtension(HvdcAngleDroopActivePowerControl.class, hvdcAngleDroopActivePowerControl);
```

Comment:
Maybe keep also the test without the extensions, and check that in that case, nothing is generated for the extensions in json files ...